### DISCUSSION TITLE
Can not merge due to GitHub keep spinning

### DISCUSSION THREAD

**Discussion by kazi-shahin** [community/community#31202] [Pull Requests]
https://drive.google.com/file/d/1fLEE4smKXrbVbqD_5XKpPLmGmUOW0lyi/view?usp=sharing

**Comment by queenofcorgis** [comment]
Hi there - are you still experiencing this? And is this in all repositories or just one?

**Comment by kazi-shahin** [comment]
It was happening due to reviews rules and I just turn off the rules and it's working
